### PR TITLE
 add migration to set audit entity set partitions to be the same as t…

### PR DESCRIPTION
**Pull request by katherinebernstein**
…he entity sets they audit

### CODE REVIEW

File: src/main/kotlin/com/openlattice/mechanic/upgrades/UpdateAuditEntitySetPartitions.kt [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+                auditRecordEntitySets.forEach {
+                    val entitySetId = it.key.first()
+
+                    if (entitySetPartitions.containsKey(entitySetId)) {
+                        val partitionsArr = PostgresArrays.createIntArray(conn, entitySetPartitions.getValue(entitySetId))
+
+                        (it.value.auditRecordEntitySetIds + it.value.auditEdgeEntitySetIds).forEach { auditEntitySetId ->
+                            ps.setArray(1, partitionsArr)
+                            ps.setObject(2, auditEntitySetId)
```
Comment (you-leee) [inline src/main/kotlin/com/openlattice/mechanic/upgrades/UpdateAuditEntitySetPartitions.kt]: Why not batch for all audit entity sets?

**Comment by you-leee** [comment]
Just a general question: Shouldn't we also update entries made for audit records? Or is that only the case for local auditing?

**Review by makosblade — APPROVED** [review]

File: src/main/kotlin/com/openlattice/mechanic/upgrades/UpdateAuditEntitySetPartitions.kt (hunk 1 above)
Comment (katherinebernstein) [reply src/main/kotlin/com/openlattice/mechanic/upgrades/UpdateAuditEntitySetPartitions.kt]: +1

**Review by katherinebernstein — COMMENTED** [review]
> Just a general question: Shouldn't we also update entries made for audit records? Or is that only the case for local auditing?

What entries?

**Comment by you-leee** [comment]
Nvm, I realized we only have S3 objects on prod and none of them are integrated yet.

File: src/main/kotlin/com/openlattice/mechanic/upgrades/Version.kt [hunk 2]
```diff
@@ -39,5 +39,6 @@ enum class Version(val value: Long) {
     V2019_08_26(11L),
     V2019_09_15(12L),
     V2019_10_03(13L),
-    V2019_11_21(14L)
+    V2019_11_21(14L),
+    V2019_12_12(15L)
```
Comment (dbt) [inline src/main/kotlin/com/openlattice/mechanic/upgrades/Version.kt]: the trailing comma is the oxford comma of programming.

**Review by dbt — APPROVED** [review]
just one comment :)

**Review by you-leee — APPROVED** [review]

**Review by pckeyes — APPROVED** [review]

File: src/main/kotlin/com/openlattice/mechanic/upgrades/Version.kt (hunk 2 above)
Comment (katherinebernstein) [reply src/main/kotlin/com/openlattice/mechanic/upgrades/Version.kt]: 😢

Comment (katherinebernstein) [reply src/main/kotlin/com/openlattice/mechanic/upgrades/Version.kt]: If we're going all in on the trailing commas, can we update our autoformatter to insert them?

**Pull request merged**